docs/conf.py

```diff
@@ -19,6 +19,7 @@
 
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
```

Comment:
```suggestion
```
See 3 lines below ;)

Reply:
The mention below is an option for autodoc and I thought I needed to include it here, when I use it ecplicitly. But it seems to work without as well so I removed it. Maybe its imported as part of autodoc in this case.

Reply:
This is the current conf.py. It's right there. ;)

```Python
extensions = [
    'sphinx.ext.autodoc',
    'sphinx.ext.viewcode',
    'sphinx.ext.napoleon',
    'sphinx.ext.autosummary', # <--- ;)
    'matplotlib.sphinxext.plot_directive',
    'sphinx.ext.mathjax',
    'sphinx.ext.intersphinx',
    'autodocsumm',
    'sphinx_design',
    'sphinx_favicon',
    'sphinx_reredirects',
    'sphinx_mdinclude',
]
```